src/main/java/de/tum/in/www1/artemis/service/connectors/localvc/LocalVCRepositoryUrl.java

```diff
@@ -0,0 +1,154 @@
+package de.tum.in.www1.artemis.service.connectors.localvc;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import de.tum.in.www1.artemis.domain.VcsRepositoryUrl;
+import de.tum.in.www1.artemis.exception.localvc.LocalVCInternalException;
+
+/**
+ * Represents a URL to a local VC repository.
+ */
+public class LocalVCRepositoryUrl extends VcsRepositoryUrl {
```

Comment:
Shouldn't this class be in the domain package just like the `VcsRepositoryUrl`?

Reply:
All the inheritors of `VcsRepositoryUrl` (e.g. `BitbucketRepositoryUrl`) are located in the package of that version control system and I think I prefer it this way, keeping everything connected to local VC in one spot.